sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -145,6 +145,13 @@ The client exposes a `begin_<method-name>` method that returns an `LROPoller` or
 for the operation to complete by calling `result()` on the operation returned from the `begin_<method-name>` method. 
 Sample code snippets are provided to illustrate using long-running operations [below](#examples "Examples").
 
+### Async APIs
```

Comment:
It doesn't feel right to have this under `Key Concepts`... Maybe at the end of Examples? Since we have a bunch of sync examples on the README then this could help point to the async ones